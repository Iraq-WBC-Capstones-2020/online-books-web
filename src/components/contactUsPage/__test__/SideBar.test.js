import React from 'react';
import renderer from 'react-test-renderer';
import ContactInformation from './../SideBar';

test('SideBar snapshot', () => {
  const tree = renderer.create(<ContactInformation />).toJSON();
  expect(tree).toMatchSnapshot();
});
