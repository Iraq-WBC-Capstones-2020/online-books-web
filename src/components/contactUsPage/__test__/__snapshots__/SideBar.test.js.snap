// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SideBar snapshot 1`] = `
<div
  className="flex justify-center mt-5"
>
  <div
    className="card max-h-full bg-blue-800 flex justify-center w-full"
    data-test="card"
  >
    <h1
      className=" my-4 pb-2 text-center white-text text-2xl"
    >
      contactUsPage.contactUs
    </h1>
    <div
      className="flex justify-start animate transition duration-500 ease-in-out   transform hover:-translate-y-1 hover:scale-110 m-4"
    >
      <i
        className="fas fa-map-marker-alt text-4xl white-text lg:ml-16"
      />
      <p
        className="text-gray-400 ml-4"
      >
        A108 Adam Street
        <br />
         New York NY 535022
      </p>
    </div>
    <div
      className="flex justify-start animate transition duration-500 ease-in-out   transform hover:-translate-y-1 hover:scale-110 m-4"
    >
      <i
        className="fas fa-envelope pr-2 text-4xl white-text lg:ml-16"
      />
      <p
        className="text-gray-400 ml-2"
      >
        contact@example.com
        <br />
        info@example.com
      </p>
    </div>
    <div
      className="flex justify-start animate transition duration-500 ease-in-out   transform hover:-translate-y-1 hover:scale-110 m-4"
    >
      <i
        className="fas fa-phone pr-2 text-4xl white-text lg:ml-16"
      />
      <p
        className="text-gray-400 ml-2 mb-24"
      >
        + 01 234 567 89 
        <br />
        +1 5589 55488 51
      </p>
    </div>
  </div>
</div>
`;
